src/dev-server/index.ts

```diff
@@ -0,0 +1,53 @@
+import _ from "lodash";
+import { spawn } from "child_process";
+import { Config } from "../config";
+import { findCwd, pipeLogsWithPrefix, waitDevServerReady } from "./utils";
+import logger = require("../utils/logger");
+import type { Hermione } from "../hermione";
+
+export type DevServerOpts = { hermione: Hermione; devServerConfig: Config["devServer"]; configPath: string };
+
+export type InitDevServer = (opts: DevServerOpts) => Promise<void>;
+
+export const initDevServer: InitDevServer = async ({ hermione, devServerConfig, configPath }) => {
+    if (!devServerConfig || !devServerConfig.command) {
+        return;
+    }
+
+    logger.log("Starting dev server with command", `"${devServerConfig.command}"`);
+
+    if (!_.isEmpty(devServerConfig.args)) {
+        logger.log("Dev server args:", JSON.stringify(devServerConfig.args));
+    }
+
+    if (!_.isEmpty(devServerConfig.env)) {
+        logger.log("Dev server env:", JSON.stringify(devServerConfig.env, null, 4));
```

Comment:
do you think all this data needs to be specified? Maybe show them by some condition? For example use `debug` library

Reply:
> do you think all this data needs to be specified?

Why not? I don't think there would be a lot of options, since it only logs **extra** env variables, which are passed directly from config file